components/BottomAppBar/src/MDCBottomAppBarViewController.h

```diff
@@ -0,0 +1,87 @@
+/*
+ Copyright 2016-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import <UIKit/UIKit.h>
+
+#import "MaterialButtons.h"
+
+/** The position of the floating action button. */
+typedef NS_ENUM(NSInteger, MDCBottomAppBarFloatingButtonPosition) {
+  MDCBottomAppBarFloatingButtonPositionCenter = 0,
+  MDCBottomAppBarFloatingButtonPositionLeading = 1,
+  MDCBottomAppBarFloatingButtonPositionTrailing = 2
+};
+
+/**
+ A material design bottom bar.
+
+ The bottom bar is a bar docked at the bottom of the screen. A floating action button is provided
+ for a primary action.
+ */
+@interface MDCBottomAppBarViewController : UIViewController
+
+/**
+ Is the floating button on the bottom bar visible.
+ Default is YES.
+ */
+@property(nonatomic, assign, getter=isFloatingButtonVisible) BOOL floatingButtonVisible;
```

Comment:
UIKit tends to lean toward isXYZHidden.
Do we want to follow that pattern?

Reply:
Done.